### PR TITLE
post v1 roadmap

### PR DESCRIPTION
**Pull request by prasek**
### Description of your changes
- added v1.1.0 release
- moved v1.0 to released

- [x] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable test` to ensure this PR is ready for review.

[skip ci]

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -29,51 +29,42 @@ Any dates listed below and the specific issues that will ship in a given milesto
...
-    * v1beta1 APIs
-
-  * Code Generation of Providers (work-in-progress)
+  * Code Generation of Providers (100% coverage)
     * AWS ACK Code Generation of the Crossplane provider-aws
-      * [initial auto generated resources](https://github.com/crossplane/provider-aws/issues/149#issuecomment-718208201)
+      * auto generate all available types in the [aws-sdk-go/models/apis](https://github.com/aws/aws-sdk-go/blob/master/models/apis)
     * Azure Code Generation of the Crossplane provider-azure
-      * [initial auto generated resources](https://github.com/matthchr/k8s-infra/tree/crossplane-hacking)
+      * auto generate all available types from the Azure metadata.
```
Comment (hasheddan) [inline ROADMAP.md:50]: nitpick:
```suggestion
      * auto generate all available types from the Azure metadata
```

**Review by hasheddan — APPROVED** [review]
LGTM!

File: ROADMAP.md (hunk 1 above)
Comment (prasek) [reply ROADMAP.md]: good catch, fixed!

File: ROADMAP.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ Any dates listed below and the specific issues that will ship in a given milesto
 ## Table of Contents
 
 * [What's next](#whats-next)
-
-  * [v1.0.0 Release Candidate (Dec 2020)](#v100-release-candidate-dec-2020)
+  * [v1.1.0 Provider Code Generation, Multi-Langauage Support, Composition Enhancements](#v110-provider-code-generation-multi-langauage-support-composition-enhancements)
```
Comment (kasey) [inline ROADMAP.md:10]: fix typo:
s/Multi-Langauage Support/Multi-Language Support/

File: ROADMAP.md [hunk 3]
```diff
@@ -29,51 +29,42 @@ Any dates listed below and the specific issues that will ship in a given milesto
 
 ## What's Next
 
-### v1.0.0 Release Candidate (Dec 2020)
+### v1.1.0 Provider Code Generation, Multi-Langauage Support, Composition Enhancements
```
Comment (kasey) [inline ROADMAP.md:32]: `Langauage` again :)

File: ROADMAP.md [hunk 4]
```diff
@@ -110,7 +101,36 @@ Any dates listed below and the specific issues that will ship in a given milesto
...
+    * v1beta1 APIs
+
+  * Code Generation of Providers (work-in-progress)
+    * AWS ACK Code Generation of the Crossplane provider-aws
+      * [initial auto generated resources](https://github.com/crossplane/provider-aws/issues/149#issuecomment-718208201)
+    * Azure Code Generation of the Crossplane provider-azure
+      * [initial auto generated resources](https://github.com/matthchr/k8s-infra/tree/crossplane-hacking)
+    * Clouds that don't have code gen pipelines
+      * Wrap stateless Terraform providers (work-in-progress) [#262](https://github.com/crossplane/crossplane/issues/262)
+      * [initial auto generated resources](https://github.com/kasey/provider-terraform-aws/tree/master/generated/resources)
```
Comment (kasey) [inline ROADMAP.md:131]: we have crossplane-contrib repos for the 3 subprojects you can start using:
https://github.com/crossplane-contrib/provider-terraform-aws
https://github.com/crossplane-contrib/terraform-runtime
https://github.com/crossplane-contrib/terraform-provider-gen

File: ROADMAP.md (hunk 2 above)
Comment (prasek) [reply ROADMAP.md:10]: thx

File: ROADMAP.md (hunk 3 above)
Comment (prasek) [reply ROADMAP.md:32]: fixed

File: ROADMAP.md (hunk 4 above)
Comment (prasek) [reply ROADMAP.md]: cool, thx 👍

**Comment by hasheddan** [comment]
@prasek would you mind giving this a quick rebase :)

**Comment by prasek** [comment]
should be good to go now

**Pull request merged**